sway/server.c

```diff
@@ -153,9 +154,15 @@ bool server_init(struct sway_server *server) {
 		return false;
 	}
 
+	server->noop_backend = wlr_noop_backend_create(server->wl_display);
+
 	struct wlr_output *wlr_output = wlr_noop_add_output(server->noop_backend);
 	root->noop_output = output_create(wlr_output);
 
+	server->headless_backend =
```

Comment:
What happens here if a headless backend was already created in `wlr_backend_autocreate`?

Reply:
Then we get two headless backends, but it's no big deal, since they share the renderer.

Reply:
Might this be a potential source of confusion for future contributors?

Reply:
I think conditionally creating the headless backend would be even more confusing.

Reply:
Added a comment to disambiguate